js/graviex.js

```diff
@@ -0,0 +1,991 @@
+        //     "accounts_filtered": [
+        //     {
+        //         "currency": "gio",
+        //         "name": "GravioCoin",
+        //         "balance": "0.0",
+        //         "locked": "0.0",
+        //         "status": "ok",
+        //         "is_purged": false,
+        //         "is_coin": true
+        //     }
+        //     ]
+        // }
+        return this.parseBalance (response);
+    }
+
+    parseDepositAddress (depositAddress, currency = undefined) {
+        const code = (currency === undefined) ? undefined : currency['code'];
+        const tag = undefined;
+        this.checkAddress (depositAddress);
+        return {
+            'currency': code,
+            'address': depositAddress,
+            'tag': tag,
+            'network': undefined,
+            'info': depositAddress,
+        };
+    }
+
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'currency': currency['id'].toLowerCase (),
+        };
+        const response = await this.privateGetDepositAddress (this.extend (request, params));
+        // "\"MJhWAUptMUrMo8Jb9G7xsPfSQjeWSWJh2q\""
+        const parsedResponse = JSON.parse (response);
+        return this.parseDepositAddress (parsedResponse, currency);
+    }
+
+    parseTransaction (transaction, currency = undefined) {
+        // [
+        //     {
+        //         "id": 4831985,
+        //         "currency": "ltc",
+        //         "amount": "0.14108256",
+        //         "fee": "0.0",
+        //         "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //         "uid": null,
+        //         "created_at": "2022-02-01T18:10:32+03:00",
+        //         "confirmations": "6",
+        //         "done_at": null,
+        //         "state": "accepted"
+        //     }
+        // ]
+        const timestamp = this.parse8601 (this.safeString (transaction, 'created_at'));
+        const id = this.safeString (transaction, 'id');
+        const txid = this.safeString (transaction, 'txid');
+        const cancelRequested = this.safeValue (transaction, 'cancel_requested');
+        const type = (cancelRequested === undefined) ? 'deposit' : 'withdrawal';
+        const amount = this.safeNumber (transaction, 'amount');
+        const currencyId = this.safeString (transaction, 'currency');
+        const confirmations = this.safeString (transaction, 'confirmations');
+        const code = this.safeCurrencyCode (currencyId);
+        let status = this.parseTransactionStatus (this.safeString (transaction, 'state'));
+        const statusCode = this.safeString (transaction, 'code');
+        if (cancelRequested) {
+            status = 'canceled';
+        } else if (status === undefined) {
+            status = this.parseTransactionStatus (statusCode);
+        }
+        return {
+            'info': transaction,
```

Comment:
this transaction structure is incorrect / incomplete

Reply:
please check the docs for the structure of a transaction

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/162b106bdb42084917d6187a3a60fb6e0f55e7d1